.moai/specs/SPEC-CACHE-ORDER-001/research.md

```
@@ -0,0 +1,214 @@
+
+| 영역 | 현재 (As-Is) | 목표 (To-Be) | 격차 |
+|------|--------------|--------------|------|
+| Static-first 표준 | 부재 | 모든 agent/skill prompt 표준화 | 신규 정책 |
+| `<system-reminder>` 사용 가이드 | 부재 | dynamic hint는 message로 전달 | 신규 가이드 |
+| 모델 전환 회피 정책 | 부분 (advisor) | 명시적 정책 + advisor 권장 | 정책 보강 |
+| Cache hit rate metric | 부재 | `.moai/metrics/cache-hit-rate.jsonl` | 신규 영속화 |
+| Prompt 구조 표준 (4 part) | 부재 | static-prefix → dynamic-suffix → reminders → user-input | 신규 |
+
+### 2.3 4-part prompt 구조 (안)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)